src/lib/fileutils.ts

```diff
@@ -194,7 +194,7 @@ export const serviceBuildAndUpdatePipeline = (
                   ``,
                   `# Update HLD`,
                   `git checkout -b "$BRANCH_NAME"`,
-                  `../fab/fab set --subcomponent $SERVICE_NAME_LOWER image.tag=$(Build.SourceBranchName)-$(Build.BuildNumber)`,
+                  `../fab/fab set --subcomponent $(Build.Repository.Name).$SERVICE_NAME_LOWER.$(Build.SourceBranchName).chart image.tag=$(Build.SourceBranchName)-$(Build.BuildNumber)`,
```

Comment:
we might need to do some massaging of sourceBranchName - what if it was a git flow branch name (ie fix/some-issue-number or dev/foo-bar-baz)

Reply:
i think in that case we'd want something that transforms / to a - ie:

release/dev -> release-dev

Reply:
it now uses a SAFE_SOURCE_BRANCH var that is `$(echo $(Build.SourceBranchName) | tr / - | tr . -)`